src/main.go

```diff
@@ -99,14 +102,21 @@ func main() {
 		endpointMetrics.StatusCode,
 		endpointMetrics.MonitorCount,
 	)
+	// go routine that reloads the entire map(cMap).
 	go func() {
-		for {
-			<-reload
-			log.Infoln("reloading...")
+		log.Infoln("Initiated monitoring your routes...")
```

Comment:
Do we need this? (the below reloading seems enough. maybe change `reloading` => `refreshing new routes in memory`)